cpu/esp_common/include/cpu.h

```diff
@@ -30,13 +30,14 @@ extern "C" {
 #define PROVIDES_PM_SET_LOWEST
 
 /**
- * @brief   Print the last instruction's address
+ * @brief   Gets the last instruction's address
  *
  * @todo:   Not supported
  */
-static inline void cpu_print_last_instruction(void)
+static inline uintptr_t cpu_get_last_instruction(void)
 {
     /* This function must exist else RIOT won't compile */
+    return 0;
```

Comment:
```suggestion
    uintptr_t r;
    asm("mov %0,a0":"=r"(r));
    return r;
```

maybe someone with an esp32 can test this

Reply:
maybe `mov.n` has to be used instead of move